test-case/check-playback.sh

```diff
@@ -94,6 +100,22 @@ do
                     dmesg > $LOG_ROOT/aplay_error_${dev}_$i.txt
                     dloge "aplay on PCM $dev failed at $i/$loop_cnt."
                     exit 1
+                else
+                    #sleep ${sleep_lst[@]} seconds before running aplay, then aplay can be run under dsp active or suspended status
+                    if [[ $(sof-dump-status.py --dsp_status 0) != "unsupported" ]] && [ $sleep -eq 1 ]; then
+                        time=${sleep_lst[${j}]}
+                        j=`expr $j + 1`
```

Comment:
`: $((j++))` is shorter, doesn't have deprecated backticks and probably cheaper/faster.

`:` has also  better compatibility with `set -e` compared to the following which aborts when using `set -e` : `j=-1; j=$(expr $j + 1)`